src/components/ChooseCategoryModal/Components/RenderCategories.tsx

```diff
@@ -0,0 +1,38 @@
+import { Image, Text, TouchableOpacity } from 'react-native';
+import React from 'react';
+import useImage from '../../../hooks/useImage';
+import { SvgXml } from 'react-native-svg';
+import { categoryIcon } from '../../../svg/svg-xml-list';
+import { useStyle } from '../styles';
+
+const RenderCategories = ({
```

Comment:
imo, it would be better to change a component name to `Categories` because a component name should be noun not verb, wdyt

Reply:
Of course. Thanks. Updated!